pkgs/by-name/rd/rdiary/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rdiary";
+  version = "0-unstable-2024-10-02";
+
+  src = fetchFromGitHub {
+    owner = "lonyelon";
+    repo = "rdiary";
+    rev = "v0.1.0";
```

Comment:
this should either match `version` or be full hash given this is unstable version